content/riak/kv/2.1.4/using/cluster-operations/backing-up.md

```diff
@@ -20,50 +20,46 @@ aliases:
 [plan backend leveldb]: /riak/kv/2.1.4/setup/planning/backend/leveldb
 [plan backend bitcask]: /riak/kv/2.1.4/setup/planning/backend/bitcask
 [use ref strong consistency]: /riak/kv/2.1.4/using/reference/strong-consistency
+[concept aae]: /riak/kv/2.1.4/learn/concepts/active-anti-entropy/
+[aae read repair]: /riak/kv/2.1.4/learn/concepts/active-anti-entropy/#read-repair-vs-active-anti-entropy
 
-Riak KV is a [clustered][concept clusters] system built to survive a wide range of
-failure scenarios, including the loss of nodes due to network or
-hardware failure. Although this is one of Riak's core strengths, it
-cannot withstand _all_ failure scenarios. Like any storage system, it
-remains susceptible to contingencies such as accidental or malicious
-deletions. Many Riak users confront this possibility by backing up their
-data, i.e. duplicating their database onto a different long-term storage
-mechanism. This document covers how to perform such backups.
-
-Riak backups can be performed using OS features or filesystems that
-support snapshots, such as LVM or ZFS, or by using tools like rsync or
-tar. Choosing your Riak backup strategy will largely depend on your
-already-established backup methodologies, as well as the backend
-configuration of your nodes. When backing up a node, it is important to
-back up the data, ring, and configuration directories of your nodes.
-
-Due to Riak's eventually consistent nature, backups can become slightly
-inconsistent from node to node. Data could exist on some nodes and not
-others at the exact time a backup is made. Any inconsistency will be
-corrected once a backup is restored, either by Riak's [active anti-entropy](/riak/kv/2.1.4/learn/concepts/active-anti-entropy/) processes or when the object is read, via [read repair](/riak/kv/2.1.4/learn/concepts/active-anti-entropy/#read-repair-vs-active-anti-entropy).
-
-Additionally, backups must be performed on a stopped node to prevent
-data loss as a result of the background merging and compaction processes
-of Riak's backends. Downtime of a node can be significantly reduced by
-using an OS feature or filesystem that supports snapshotting.
+Riak KV is a [clustered][concept clusters] system built to survive a wide range of failure scenarios including the loss of nodes due to network or hardware failure. Although this is one of Riak KV's core strengths, it cannot withstand all failure scenarios.
+
+Backing up data; duplicating the database on a different long-term storage system, is a common approach to mitigating potential failure scenarios.
```

Comment:
"Backing up data (duplicating the database on a different long-term storage system)..."